### ISSUE TITLE
[Feature] Using  Playwright to automate some of the guided tests for accessibility Insights for Web 

### ISSUE THREAD

**Issue by harir5** [microsoft/playwright#13338]
Hi,

Just a thought at the moment. Accessibility insights for web(which i stumbled upon in your community showcase page) currently offers some guided tests and automated checks; the guided tests that are provided have to be run manually( for instance keyboard navigation and tabbing) - is there a way to run some playwright tests (for instance number of tabs can be provided as input for a particular page and the script can tab across different components in the page) and at the end pass or fail the accessibility test perhaps ? 

is this feasible?

**Comment by harir5** [comment]
@mxschmitt - checks like these :
![image](https://user-images.githubusercontent.com/28225178/161939568-3adc4cbb-4054-4e6e-84f7-c598e5e4ab5e.png)

this 'guided' test has to be run manually and the user then has to pass/fail the test manually. If we can automate this in Playwright and pass/fail the guided test from code, it would really be useful during accessibility assessments.

**Comment by mxschmitt** [comment]
I'll mark it as a feature request, so we can include it into the accessibility story when we revisit the accessibility topic.

**Comment by dbjorge** [comment]
Hi @harir5 ! I'm an engineer on the Accessibility Insights team.

Our recommendation for folks that want to include the automatable portions of Accessibility Insights in a Playwright test suite is to use [@axe-core/playwright](https://www.npmjs.com/package/@axe-core/playwright) to write test cases using [axe-core](https://github.com/dequelabs/axe-core), the accessibility rules engine that Accessibility Insights uses for its automated checks.

I've just written up a feature proposal (#14112) for adding official documentation to playwright.dev explaining how to do this, but in the meantime, our team maintains some samples at [microsoft/axe-pipelines-samples](https://github.com/microsoft/axe-pipelines-samples/tree/main/typescript-playwright-sample#readme) that might give you a good starting point.

**Comment by harir5** [comment]
@dbjorge - thank you for your response. I have already completed a small proof of concept with Axe-Core. is it possible to get in touch with you to discuss about this feature kindly ?

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your involvement. This issue was closed due to limited engagement (upvotes/activity), lack of recent activity, and insufficient actionability. To maintain a manageable database, we prioritize issues based on these factors.

If you disagree with this closure, please open a new issue and reference this one. More support or clarity on its necessity may prompt a review. Your understanding and cooperation are appreciated.
